src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -946,14 +984,14 @@ void ImportCall(ILOpcode opcode, int token)
                 }
             }
 
+            // Check any constraints on the callee's class and type parameters
+            var ecmaType = method.OwningType as EcmaType;
+            if (!method.OwningType.CheckConstraints())
+                VerificationError(VerifierError.UnsatisfiedMethodParentInst, method.OwningType);
+            else if (!method.CheckConstraints())
```

Comment:
The reason why I added this else, is because `method.CheckConstraints` of the ConstraintsHelper also checks whether the method's owning type's constraints are satisfied, which then reports an "UnsatisfiedMethodParentInst" *and* an "UnsatisfiedMethodInst" error, even though only the method parent was invalid.
With the current version an "UnsatisfiedMethodInst" error is only reported once the "UnsatisfiedMethodParentInst" error is fixed...
Do you still prefer to always check both?

Reply:
I see. Ignore my comment then.